install/operator/pkg/generator/assets/infrastructure/03-syndesis-server-config.yml.tmpl

```diff
@@ -65,7 +65,11 @@
         jaeger-activity-tracing:
           enabled: true
       jaeger:
+      {{- if .Syndesis.Addons.Jaeger.QueryUri}}
+        query.api.uri: {{.Syndesis.Addons.Jaeger.QueryUri}}
```

Comment:
The property name should end with `url` and not `uri`. WDYT to use `uri` as standard, instead of the `url` ?